common/men_at_arms_types/00_maa_types.txt

```diff
@@ -125,7 +130,36 @@ pikemen_unit = {
 	ai_quality = { value = culture_ai_weight_pikemen }
 	icon = pikemen
 }
+###WARCRAFT###
+caster_unit = {
+	type = caster
+	
+	damage = 30
+	toughness = 9
+	pursuit = 10.5
```

Comment:
Pursuit isn't visible as a float number.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103152987-5f109e80-47a6-11eb-862b-113fac2ce00f.png)

</details>